### DISCUSSION TITLE
Error during installation.

### DISCUSSION THREAD

**Discussion by hohemathaus** [Budibase/budibase#2052] [Help]
Hi good people...Iam getting this error in my installation.
maza@maza:~$ sudo budi hosting --start
[sudo] password for maza:
Starting services, this may take a moment - first time this may take a few minut                                                                                                                es to download images.
Creating network "maza_default" with the default driver
Creating volume "maza_couchdb3_data" with local driver
Creating volume "maza_minio_data" with local driver
Creating volume "maza_redis_data" with local driver
Pulling minio-service (minio/minio:)...
latest: Pulling from minio/minio
error pulling image configuration: Get https://registry-1.docker.io/v2/minio/min                                                                                                                io/blobs/sha256:a72ace24f9cb97bf85f2e6ff6e21a3f39f55d832b1f2fa71223580bf700166c4                                                                                                                : dial tcp: lookup registry-1.docker.io on 127.0.0.53:53: read udp 127.0.0.1:600

**Comment by mike12345567** [comment]
Duplicate of #2047
